crates/config/src/lib.rs

```diff
@@ -1093,7 +1102,15 @@ impl Config {
         maybe_alias: &str,
     ) -> Option<Result<Cow<'_, str>, UnresolvedEnvVarError>> {
         let mut endpoints = self.rpc_endpoints.clone().resolved();
-        Some(endpoints.remove(maybe_alias)?.map(Cow::Owned))
+        if let Some(endpoint) = endpoints.remove(maybe_alias) {
+            return Some(endpoint.map(Cow::Owned))
+        }
+
+        if let Ok(Some(endpoint)) = mesc::get_endpoint_by_query(maybe_alias, Some("foundry")) {
```

Comment:
We're ignoring the Err variant here, is this right? @onbjerg @klkvr

Reply:
there are errors in there related to eg mesc not being enabled and such, which is why i chose to ignore it. don't want it to fail for people not using mesc